src/configCors.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Request, Response } from "express";
+
+// eslint-disable-next-line consistent-return
+const configCors = (req: Request, res: Response, next: NextFunction) => {
+  res.header("Access-Control-Allow-Origin", "*");
```

Comment:
David mirate en el SonarCloud lo del " 1 HotSpot Security" => no le mola nada "*" si no entiendo mal es que entre todo el mundo y se deberia limitar.  Hay una pestañita que pone "How can you fix, it?" que lo explica. Seguro que lo entiendes mejor que yo

Reply:
Sip lo tengo visto, tengo que acabar de mirarme la documentación. Aunque probablemente al final acabemos usando al libreria de cors instalandolo de npm y viene ya configurado sin problemas.

Reply:
si algien ya lo ha programado es buena